addons/web/static/lib/owl/owl.js

```diff
@@ -3660,16 +3660,28 @@
             this.vnode = component.__owl__.vnode || h("div");
             const qweb = component.env.qweb;
             let root = component;
-            let canCatch = false;
-            while (component && !(canCatch = !!component.catchError)) {
-                root = component;
-                component = component.__owl__.parent;
-            }
-            qweb.trigger("error", error);
-            if (canCatch) {
-                component.catchError(error);
+            function handle(error) {
+                let canCatch = false;
+                qweb.trigger("error", error);
+                while (component && !(canCatch = !!component.catchError)) {

```

Comment:
not common, and in general not encouraged, but it is assigned in the while condition